.changeset/poor-dragons-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@openfun/cunningham-react": minor
+---
+
+improve Pagination behavior
```

Comment:
Don't know if it's related to this commit but i've an error on the pagination story that is not on main.
![image](https://github.com/openfun/cunningham/assets/139848/ef22970a-5cf5-4d2b-a7d3-826d568bb3a6)